src/utils/pageRouter/isTokenValid.js

```
@@ -1,12 +1,12 @@
-import jwt_decode from 'jwt-decode';
+import {jwtDecoder, TokenPart} from '@kinde/jwt-decoder';
 import {config} from '../../config/index';
 
 const isTokenValid = (token) => {
   const accessToken = (token && token.access_token) || token;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)